knowing-more-about-ourselves.rst

```diff
@@ -0,0 +1,206 @@
+suggested using `Metabase <https://www.metabase.com/>`_ since they were already using it for a similar use case and they were happy with it.
+We connected our PostgreSQL replica database to it and started plotting this data to get some insights.
+We found awesome things we want to start sharing with all the documentation community,
+since we think it will useful not only for ourselves but also for package maintainers.
+
+Sphinx extensions used in the last 15 days
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/sphinx-extensions.png
+   :align: center
+   :width: 100%
+
+
+This plot helps us to understand the Sphinx ecosystem in general and
+to measure the adoption of our own extensions:
+`sphinx-hoverxref <https://sphinx-hoverxref.readthedocs.io/>`_  and `sphinx-notfound-page <https://sphinx-notfound-page.readthedocs.io/>`_.
+We plan to share these results to the Sphinx maintainers,
+since this will be helpful on the conversation we are having about `Removing JavaScript Dependencies <https://github.com/sphinx-doc/sphinx/issues/10070>`_.
+
+.. note::
+
+   We excluded our own theme and Sphinx's core depedencies because they are too high and break the relation with the others in the graph.
+   Also, only extensions used by more than 10 projects are showed.
+
+Sphinx adoption over time
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/sphinx-adoption.png
+   :align: center
+   :width: 100%
+
+
+On this plot we can see that with each new release of Sphinx,
+the old versions decrease while the just released one increases.
+This is mainly because most of the projects are not pinning Sphinx and Read the Docs is installing the latest version by default.
+Note that versions marked with a red rectangle are the most relevant ones for this analysis,
+and are highlighted just to improve readability.
+
+There is something that's pretty prominent on this plot as well.
+The old version 1.8.6 has around 1000 projects, which could look suspicious at first sight.
+However, this is because Read the Docs still defaults to 1.8.6 on projects created before Oct 20, 2020.
+This is to keep compatibility with old projects and avoid breaking their builds without notification.
+
+
+Projects using MyST to write Markdown in Sphinx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/myst-parser.png
+   :align: center
+   :width: 100%
+
+
+This is a similar plot than the previous one but for `MyST parser <https://myst-parser.readthedocs.io/>`_.
+We are interested in knowing how many projects are writing Markdown using Sphinx.
+Many people don't know it's possible to write Markdown while keep using Sphinx and all the amazing features it has.
+Interestingly, the number of projects using MyST parser, currently, is higher than the number of projects using MkDocs.
+If you are reading this and you didn't know about MyST parser,
+we strongly recommend you to give it a try!
```

Comment:
This is a good place for a link to a guide on using MyST

Reply:
I put the link at the beginning of this paragraph already. Should I link this last sentence to the "Get started" guide? https://myst-parser.readthedocs.io/en/latest/intro.html